README.md

```diff
@@ -1,47 +1,322 @@
+
+    ```bash
+    source .venv/bin/activate
+    ```
+
+### ⚙️ Local Configuration
+
+Configure all required settings in a local `settings.conf` file within the `packages/automated_actions` directory (or wherever your main application package expects it). Example:
+
+```ini
+# packages/automated_actions/settings.conf (example)
+AA_DEBUG=1
+AA_WORKER_METRICS_PORT=8001
+AA_OIDC_CLIENT_ID=<RHSSO_CLIENT_ID>
+AA_OIDC_CLIENT_SECRET=<RHSSO_CLIENT_SECRET>
+# ... other AA_ variables ...
+AA_ENVIRONMENT="local-dev"
+
+OPA_ACTION_API_URL=http://localhost:8080/api/v1 # Adjust if your local API runs elsewhere
+OPA_ACTION_API_TOKEN=<OPA_ACTION_API_TOKEN>
+OPA_MAX_AGE_MINUTES=10
+```
+
+Ensure that the `AA_*` variables are set according to your local environment. The `OPA_ACTION_API_TOKEN` can be generated using the `create-token` command provided by the `automated_actions_cli` (once available and configured):
 
 ```bash
-automated-actions
+automated-actions create-token --name "OPA service account" --username "open-policy-agent" --email "not-used@example.com" --expiration "2200-09-30 20:15:00"
 ```
 
-## Development
+Ensure the username is `open-policy-agent` and the expiration date is set to a future date. This command will return a token that you can use in your local configuration.
 
-Create and maintain a development virtual environment via:
+### ▶️ Running the Application (Locally)
+
+Use the provided `docker-compose.yml` file at the root of the project to run the application and its dependencies (like LocalStack for AWS services) locally.
 
 ```bash
-make dev-env
+docker-compose up automated-actions # Or 'docker-compose up -d' for detached mode
 ```
 
-Activate the virtual environment in the current shell via:
+This will typically set up the FastAPI server, Celery workers, and mock AWS services. Check the `docker-compose.yml` for specific service names and configurations.
+
+### 🔬 Testing
+
+Run the main test suite from the root directory:
 
 ```bash
-source .venv/bin/activate
+make test
 ```
 
-Run the tests suite via:
+Each package may also have its own test suite. To run tests for a specific package:
 
 ```bash
-make tests
+cd packages/automated_actions
+make test
 ```
 
-[Api docs](http://localhost:8080/redoc)
-[Swagger](http://localhost:8080/doc)
+### Linting and Formatting
+
+This project uses [Ruff](https://github.com/astral-sh/ruff) for linting and formatting. To check for issues:
+
+```bash
+make format
+```
+
+### Release
+
+`automated-actions-client` and `automated-actions-cli` are released to PyPI. To release a new versions:
+
+1. Recreate the `automated-actions-client` package after changes to the OpenAPI spec.
+
+    ```bash
+    make generate-client
+    ```
+
+1. Update the version numbers in [automated_actions_client/pyproject.toml](/packages/automated_actions_client/pyproject.toml). [automated_actions_cli/pyproject.toml](/packages/automated_actions_cli/pyproject.toml).
+1. Update the changelog in [CHANGELOG.md](/CHANGELOG.md).
```

Comment:
remove this since we don't have changelog

Reply:
Removed in [9ea0803](https://github.com/app-sre/automated-actions/pull/120/commits/9ea0803984554bac1dc238155cdde0c5e20c2307)